### DISCUSSION TITLE
General Project Discussion

### DISCUSSION THREAD

**Discussion by Deeds101** [Deeds101/CYBR8420-project#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DoomDragoon** [comment]
Hey everyone, When you are satisfied with your diagram, move a copy into the final diagram folder and we will merge it with our Markdown doc when everyone has theirs ready.
